### DISCUSSION TITLE
How to Install on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by norsemangrey** [lsd-rs/lsd#941] [Q&A]
Linux newbie here. Can anyone please help me with an explanation on how to install the latest version of lsd on Ubuntu 22.04?

**Comment by zwpaper** [comment]
did you try installing the deb? Or meet this ? https://github.com/lsd-rs/lsd/issues/891

**Comment by stkania** [comment]
You can get via snap "sudo snap install lsd"

**Reply by zwpaper** [reply]
nope, snap is not recommended, please use deb or binary directly.
